example/index.html

```diff
@@ -138,7 +158,7 @@
 
       function getPlayerConfig() {
         return {
-          key: 'YOUR-PLAYER-KEY',
+          key: '<playerKey>',
```

Comment:
We should probably use `<PLAYER_LICENSE_KEY>` like we do in https://developer.bitmovin.com/playback/docs/getting-started-web and other places for the sake of consistency

```suggestion
          key: '<PLAYER_LICENSE_KEY>',
```

Reply:
https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/109/commits/9117ee4137992e61b45a9dd482ab531b830bd928